tools/topology/topology2/cavs/cavs-mixin-mixout-hda.conf

```diff
@@ -2,6 +2,16 @@ Define {
        ANALOG_PLAYBACK_PCM	'Analog Playback'
        ANALOG_CAPTURE_PCM	'Analog Capture'
        HDA_ANALOG_DAI_NAME      'Analog Playback and Capture'
+       DEEP_BUFFER               1
```

Comment:
If you set this here in the conf file, you will always end up including the deep-buffer PCM. You should move this to CMakeLists.txt

Reply:
thanks, done